ontology/yaml/resources/LIGHTING/entity_types/ABSTRACT.yaml

```diff
@@ -56,4 +56,10 @@ BM:
   uses:
   - battery_charge_status
 
+IS:
```

Comment:
Nit: can we change this to IM to match previous abstract type naming conventions, and update the description to "Individual illuminance sensor monitoring, not used for control"?

Reply:
Sure, changes made.